src/util/narrow.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Diffblue Limited. All Rights Reserved.
+
+#ifndef CPROVER_UTIL_NARROW_H
+#define CPROVER_UTIL_NARROW_H
+
+#include "invariant.h"
+#include <type_traits>
+
+/// Alias for static_cast intended to be used for numeric casting
+/// Rationale: Easier to grep than static_cast
+template <typename output_type, typename input_type>
+output_type narrow_cast(input_type value)
```

Comment:
If this is derived from Microsoft GSL this might need an appropriate copyright notice to that effect

Reply:
Not necessary, I wrote it from scratch in test-gen.